### DISCUSSION TITLE
CitraVR (Beta) v0.4.2

### DISCUSSION THREAD

**Discussion by amwatson** [amwatson/CitraVR#59] [Announcements]
## What's Changed
* Added hotkey for "quit to menu" under Settings > Gamepad

**Full Changelog**: https://github.com/amwatson/CitraVR/compare/v0.4.1...v0.4.2

<hr /><em>This discussion was created from the release <a href='https://github.com/amwatson/CitraVR/releases/tag/v0.4.2'>CitraVR (Beta) v0.4.2</a>.</em>

**Comment by KeyDesh** [comment]
Hello, how are you, I have a question not directly regarding Citra VR but related to it, if I'm going against the rules I apologize. Is it possible to map the Yuzu NCE emulator with the Quest 2 controls like in Citra VR? (use the D-PAD or analogue, X, Y, A, B, L, R...), only a programmer can do this?, how can I do it?
